packages/slate/src/controllers/editor.js

```diff
@@ -582,11 +582,16 @@ function normalizeDirtyPaths(editor) {
  * @param {Array} path
  */
 
-function normalizeNodeByPath(editor, path) {
+function normalizeNodeByKey(editor, key) {
```

Comment:
light recommendation: I believe there's a naming convention for functions that can accept a path or a key interchangably (I believe this is driven by getNode operating on either). If that's the case then I think we should borrow on that naming convention for this function.

Reply:
Ah, I may have actually broken this functions ability to work with either based on some other code I removed. I can add that back in though if it makes sense for this to accept either a path or a key.